### PR TITLE
Corrects tile color defines

### PR DESCRIPTION
**Pull request by ShizCalev**
They didn't account for the shading, leading to minute differences in shade. This corrects them so that they are indistinguishable from the currently existing plating colors.

Also corrects the layering of the decal, as well as an incorrect icon_state on one of the subtypes.

### CODE REVIEW

File: code/__DEFINES/colors.dm [hunk 1]
```diff
@@ -1,38 +1,39 @@
...
-#define COLOR_PALE_BLUE_GRAY   "#8BBBD5"
-#define COLOR_PALE_GREEN_GRAY  "#AED18B"
-#define COLOR_PALE_RED_GRAY    "#CC9090"
-#define COLOR_PALE_PURPLE_GRAY "#BDA2BA"
-#define COLOR_PURPLE_GRAY      "#A2819E"
+#define COLOR_BLUE_GRAY        "#75A2BB"
+#define COLOR_BROWN            "#BA9F6D"
+#define COLOR_DARK_BROWN       "#997C4F"
+#define COLOR_DARK_ORANGE      "#C3630C"
+#define COLOR_GREEN_GRAY       "#99bb76"
```
Comment (deathride58) [inline code/__DEFINES/colors.dm]: Capitalization is inconsistent with the rest of the defines here.

Literally unplayable

File: code/game/objects/effects/decals/turfdecal/tilecoloring.dm [hunk 2]
```diff
@@ -1,6 +1,8 @@
 /obj/effect/turf_decal/tile
 	name = "tile decal"
 	icon_state = "corner_white_full"
+	layer = TURF_PLATING_DECAL_LAYER
```
Comment (deathride58) [inline code/game/objects/effects/decals/turfdecal/tilecoloring.dm]: Layer isn't passed to the decal component. You'll have to add in a `, layer` to line 42 of decal.dm for this to have any sort of ingame effect.

**Review by Cyberboss — CHANGES_REQUESTED** [review]
@deathride58

File: code/game/objects/effects/decals/turfdecal/tilecoloring.dm (hunk 2 above)
Comment (ShizCalev) [reply code/game/objects/effects/decals/turfdecal/tilecoloring.dm]: This wasn't intended to have ingame effect, only correcting the visuals in DM.

File: code/game/objects/effects/decals/turfdecal/tilecoloring.dm [hunk 3]
```diff
@@ -173,7 +187,6 @@
 
 /obj/effect/turf_decal/tile/white
 	name = "white tile decal"
-	icon_state = "corner_white"
```
Comment (Cyberboss) [inline code/game/objects/effects/decals/turfdecal/tilecoloring.dm]: ?

Comment (ShizCalev) [reply code/game/objects/effects/decals/turfdecal/tilecoloring.dm]: op - "as well as an incorrect icon_state on one of the subtypes."

Incorrect var that wasn't corrected in the original PR.

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**